docker-storage-setup.sh

```diff
@@ -17,7 +17,7 @@
 #++
 
 #  Purpose:  This script grows the root filesystem and sets up LVM volumes
-#            for docker metadata and data.
+#            for container runtime metadata and data.
```

Comment:
Can we change it. We have deprecated old mode of setting up metadata and data volumes. So we probably can say `sets up storage for container runtime`